components/icons/images/tier2/edit-not-editable.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+  <defs>
+    <clipPath id="a">
+      <path d="M0 0h24v24H0z"/>
+    </clipPath>
+  </defs>
+  <g clip-path="url(#a)" data-name="24x24-no-editing-icon">
+    <path fill="#fff" d="M0 0h24v24H0z"/>
+    <path fill="#494c4e" stroke="#565a5c" stroke-linecap="round" stroke-width="2" d="M2.5 4.5l17 17" data-name="Path 3554" opacity=".998"/>
+    <g data-name="Group 5079">
```

Comment:
I think all the `data-name` attributes can be removed from all of these files -- they look like leftover friendly assets from whatever image editing program was used to create these.

Reply:
I'm not too knowledgeable with SVGs, but that makes sense to me